osi_lane.proto

```diff
@@ -817,6 +816,45 @@ message LaneBoundary
         // See \c LaneBoundary .
         //
         optional double height = 3;
+
+        // Alternation of dashes in case of a dashed lane boundary. In
+        // context, this field gives information about the location of
+        // dashes on the boundary line.
+        //
+        optional Dash dash = 4;
+
+        // This enum describes the alternation of dashes in case of a
+        // dashed lane boundary.
+        //
+        // \note The enum descriptions adhere to the definition direction
+        // of the lane boundary points, meaning that start or end of a dash
+        // are understood with respect to the direction in which the points
+        // of the boundary line are defined.
+        //
+        enum Dash
+        {
+            // The current state of the dash alternation is not known (must
+            // not be used in ground truth). 
```

Comment:
The current state of the dash alternation is not known and therefore must not be used in ground truth.

Reply:
I don't think this fully captures the meaning (note that those comments are present in all enums, so probably should not be adjusted as part of this PR anyway): The state is unknown. Incidentally this kind of value must not be used in ground truth, but in other areas it can be used. The therefore seems too causal to me.